extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -23,4 +23,5 @@ amp-sticky-ad {
   width: 100%;
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
+  font: 0/0 a; /* remove the gap between inline(-block) elements */
```

Comment:
This is not a good idea. We need to understand better the reason and adjust styling accordingly. I think this would probably go away if we style `amp-sticky-ad` via flexbox. This is a good idea in general as it'd center the ad.